packages/@react-stately/virtualizer/src/Virtualizer.ts

```diff
@@ -865,7 +865,7 @@ export class Virtualizer<T extends object, V, W> {
     // Apply layout infos to visible views
     for (let view of this._visibleViews.values()) {
       let cur = view.layoutInfo;
-      if (cur) {
+      if (cur.key != null) {
```

Comment:
```suggestion
      if (cur?.key != null) {
```
It looks like you found an issue with `cur.key` being `null`, but don't we still need to deal with `cur` being `null`?